scripts/ci/build_emscripten.sh

```diff
@@ -40,12 +40,9 @@ else
 	BUILD_DIR="$1"
 fi
 
-apt-get update
-apt-get install lz4 --no-install-recommends
+WORKSPACE="${2:-/root/project}"
```

Comment:
I'd just remove the default. The directory is pretty arbitrary so if we have to specify it for one case, I think it's cleaner to specify it in all cases.

Reply:
The default folder is needed by the CI jobs. The problem is that the CI runs as the root user. However, this is unnecessary locally and messes up with the permissions in the solidity folder when developing.
Maybe we could just specify `/project` for all then, and only then will it be different depending on the environment.
Alternatively, we can run the CI with a default user (e.g., 1000), assuming all base images have a user or that we created a builder user for this case.

Reply:
So I changed everything to `/project` and I'll update the checkout step in the CI to use it as the working directory.